.changeset/fluffy-cheetahs-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saleor/app-sdk": minor
+---
+
+Added `handlers/fetch-api` which adds support for frameworks that use [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
```

Comment:
please remember to add full changeset and docs

Reply:
https://github.com/saleor/saleor-docs/pull/1437